src/ensembl/src/content/app/species/components/species-stats/SpeciesStats.tsx

```diff
@@ -81,12 +83,17 @@ const SpeciesStats = (props: SpeciesStatsProps) => {
 
   return (
     <div className={styles.wrapper}>
-      <span className={styles.preLabel}>{props.preLabel}</span>
+      {props.preLabel && (
+        <span className={styles.preLabel}>{props.preLabel}</span>
+      )}
 
       <div className={styles.label}>{props.label}</div>
 
       <div>
-        <span className={styles.primaryValue}>{props.primaryValue}</span>
+        <span className={styles.primaryValue}>
+          {props.primaryValue}
+          {props.primaryValuePostfix || null}
```

Comment:
no need for `|| null`

Reply:
Removed the prop `primaryValuePostfix` as mentioned in https://github.com/Ensembl/ensembl-client/pull/352#discussion_r503607775